core/src/saros/account/XMPPAccountStore.java

```diff
@@ -594,7 +566,7 @@ public XMPPAccount findAccount(String jidString) {
    * save the result in the user home directory which should be default only be accessible by the
    * user itself or administrators.
    */
-  private static class Crypto {
+  static class Crypto {
```

Comment:
Why did you remove private ?

Reply:
For testing purposes. Otherwise it is not possible to test the accountFile format.